digid_eherkenning/views/eherkenning.py

```diff
@@ -7,19 +7,30 @@
 from django.utils.translation import gettext as _
 from django.views.generic.base import TemplateView, View
 
+from ..choices import AssuranceLevels
 from ..exceptions import SAML2Error, eHerkenningNoRSINError
 from ..forms import SAML2Form
-from ..saml2.eherkenning import (
-    eHerkenningClient,
-    generate_dienst_catalogus_metadata,
-    generate_eherkenning_metadata,
-)
+
+# These two unused imports may be imported from here by 3rd party client code
+# ignoring F401 for now
+from ..saml2.eherkenning import generate_dienst_catalogus_metadata  # noqa: F401
+from ..saml2.eherkenning import generate_eherkenning_metadata  # noqa: F401
+from ..saml2.eherkenning import eHerkenningClient
 from .base import get_redirect_url
 
 
 class eHerkenningLoginView(TemplateView):
     template_name = "digid_eherkenning/post_binding.html"
 
+    def get_level_of_assurance(self) -> Optional[AssuranceLevels]:
+        """
+        Override the AssuranceLevel from the global `EherkenningConfiguration`.
+
+        To use the level from the global config, return `None`
+        NB When overriding this, remember the user has control over the request!
+        """
+        return None
```

Comment:
needs to update to `return ""` if you decide to apply the suggested type changes above.

Reply:
:+1: and a different return type.